### PR TITLE
added assignent.rb

### PR DESCRIPTION
**Pull request by GreggDurishan**
(3rd milennium BC Iranian proto-animation, showing the phases of a goat leaping to nibble some plants, executed in successive images adorning a vase)

Added main codefile, assignment.rb
assignment.rb should currently meet the design requirements

### CODE REVIEW

**Comment by GreggDurishan** [comment]
After creating this pull req, I went to doublecheck my work again, and when viewing the assignment.rb file within the ged-development branch, the file appeared to be empty. I marked the pullreq as invalid and set about investigating. The code I expected to check in is present and represented in the assignment.rb file associated with the pull req. I retried evaluating the contents of the ged-development branch a moment later and the file contents were present as expected, so I removed the invalid label on the pull req. I'm chalking such up to a timing fluke, as things are looking as expected now.

File: assignment.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
+puts "What's your name?"
+name = ""
+while (name=="") do
```
Comment (rpearce) [inline assignment.rb]: This line works perfectly well, but I recommend going more with the ruby style that looks like this:
```ruby
while name == "" do
```
Or if you want to get fancy,
```ruby
while !!name do
```
Look up `ruby bang bang` to see what it does! Try it in IRB.

File: assignment.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	name = gets.chomp
+	if (name=="")
```
Comment (rpearce) [inline assignment.rb]: [same as above](https://github.com/GreggDurishan/1.1-user-input-ged/pull/1/files#r32690849)

File: assignment.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	end
+end
+
+puts "Please select one of the following 4 legal situations:"
+puts " 1 - I am law abiding!"
+puts " 2 - I make sure others obey the law."
+puts " 3 - I live outside the law."
+puts " 4 - Questionable."
+legalSit = ""
+while (legalSit=="") do
```
Comment (rpearce) [inline assignment.rb]: [same as above](https://github.com/GreggDurishan/1.1-user-input-ged/pull/1/files#r32690849)

**Comment by rpearce** [comment]
Best gif I've seen in a while (and thanks for the explanation).

Apart from the small `while` style comments, this is :+1:. You can make the recommended change(s) if you so desire, and close and merge at your convenience.

**Comment by GreggDurishan** [comment]
Thanks for the great feedback, I'll get right onto it! =) Style is so hard
to have foresight on!

**Pull request merged**